src/pages/HomePage/HomePage.css

```diff
@@ -0,0 +1,39 @@
+.homepage-wrapper {
+  margin: 0 5rem;
+}
+
+.homepage-main-section {
+  flex-grow: 1;
+  margin: 1.5rem 0;
+}
+
+.homepage-search-input-wrapper {
+  border: 3px solid #d1d1d1;
```

Comment:
you can define this as a variable

Reply:
I have used this color only here, so that's why I haven't created a variable for that.